example for self-hosted runners in the wild (#9015)
github/docs: content/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners.md

actions.runner.octo-org-octo-repo.runner01.service
```

> If that fails due to the service being installed elsewhere, the service name can be obtained from the list of running services. For example:

Reviewer: I'd suggest a slight change here:
```suggestion
If this fails due to the service being installed elsewhere, you can find the service name in the list of running services. For example, on most Linux systems you can use the `systemctl` command:
```

```shell
$ systemctl --type=service | grep actions.runner

>   actions.runner.org-repo.hostname.service loaded active running GitHub Actions Runner (org-repo.hostname)

Reviewer: For consistency, I think it would be better to stick to `octo-org` and `octo-repo` as the example organization and repository names throughout:

```suggestion
actions.runner.octo-org-octo-repo.hostname.service loaded active running GitHub Actions Runner (octo-org-octo-repo.hostname)
```

```

You can use `journalctl` to monitor the real-time activity of the self-hosted runner:

```shell

> # if the service name is known:

Reviewer: ```suggestion
```

> You can use `journalctl` to monitor the real-time activity of the self-hosted runner:
> 
> ```shell
> # if the service name is known:
> $ sudo journalctl -u actions.runner.octo-org-octo-repo.runner01.service -f
> 
> # if the service name is not known (look for a service name starting with "actions.runner"):
> $ sudo journalctl -u $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1) -f
> ```

Reviewer: I'd prefer to pull the comments out into normal text:

````suggestion
You can use `journalctl` to monitor the real-time activity of the self-hosted runner. For example, if you know the service name:

```shell
$ sudo journalctl -u actions.runner.octo-org-octo-repo.runner01.service -f
```

If you don't know the service name, look for a service name starting with "actions.runner":

```shell
$ sudo journalctl -u $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1) -f
```
````

> 
> # if the service name is not known (look for a service name starting with "actions.runner"):
> $ sudo journalctl -u $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1) -f

Reviewer: ```suggestion
```

Check that the self-hosted runner's service account has permission to use the Docker service. You can identify this account by checking the configuration of the self-hosted runner in systemd. For example:

```shell
# if the service name is known:
$ sudo systemctl show -p User actions.runner.octo-org-octo-repo.runner01.service
User=runner-user

> 
> # if the service name is not known (look for a service name starting with "actions.runner"):
> $ sudo systemctl show -p User $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1)
> User=runner-user

Reviewer: ```suggestion
```

> Check that the self-hosted runner's service account has permission to use the Docker service. You can identify this account by checking the configuration of the self-hosted runner in systemd. For example:
> 
> ```shell
> # if the service name is known:
> $ sudo systemctl show -p User actions.runner.octo-org-octo-repo.runner01.service
> User=runner-user
> 
> # if the service name is not known (look for a service name starting with "actions.runner"):
> $ sudo systemctl show -p User $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1)
> User=runner-user
> ```

Reviewer: As above, move the comments into body text:

````suggestion
Check that the self-hosted runner's service account has permission to use the Docker service. You can identify this account by checking the configuration of the self-hosted runner in systemd. For example, if you know the service name:

```shellfails due to
$ sudo systemctl show -p User actions.runner.octo-org-octo-repo.runner01.service
User=runner-user
```

If you don't know the service name, look for a service name starting with "actions.runner":

```shell
$ sudo systemctl show -p User $(systemctl --type=service | grep actions.runner | grep -o "[^ ]*" | head -n 1)
User=runner-user
```
````
